qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/files/FilesFragmentUITest.java

```diff
@@ -134,14 +123,27 @@ private void setupData() throws IOException, QblStorageException {
         mBoxHelper.setActiveIdentity(testIdentity);
         mBoxHelper.getService().addContact(testContact2);
 
-        mBoxHelper.setActiveIdentity(testIdentity2);
+        mBoxHelper.setActiveIdentity(testIdentity);
+
+        mBoxHelper.createFolder(TEST_FOLDER, testIdentity, "");
 
         for (ExampleFile exampleFile : exampleFiles) {
             mBoxHelper.uploadFile(mBoxHelper.mBoxVolume, exampleFile.getName(), exampleFile.getData(), "");
         }
         mBoxHelper.waitUntilFileCount(exampleFiles.size());
     }
 
+    @AfterClass
+    public static void cleanUpData() throws QblStorageException {
+        mBoxHelper.deleteFolder(CREATE_FOLDER_TEST_NAME, testIdentity, "");
+        mBoxHelper.deleteFolder(TEST_FOLDER, testIdentity, "");
+        for (ExampleFile exampleFile : exampleFiles) {
+            mBoxHelper.deleteFile(InstrumentationRegistry.getContext(), testIdentity, exampleFile.getName(), "");
+        }
+        mBoxHelper.deleteIdentity(testIdentity);
+        mBoxHelper.deleteIdentity(testIdentity2);
```

Comment:
There is no need to delete those files. Just remove the identities. Other tests can't depend on a clean environment anyway.